docker/Dockerfile-ubuntu-cuda-9.0

```diff
@@ -1,33 +1,10 @@
-FROM nvidia/cuda:9.0-devel-ubuntu16.04
+FROM nvidia/cuda:9.2-devel-ubuntu18.04
 ENV DEBIAN_FRONTEND noninteractive
-RUN apt-get update && apt-get install -y \
-    apt-utils \
-    build-essential \
-    curl \
-    lcov \
-    pkg-config \
-    openmpi-bin libopenmpi-dev \
-    libfftw3-dev \
-    libboost-dev libboost-serialization-dev libboost-mpi-dev libboost-filesystem-dev libboost-test-dev \
-    libgsl-dev \
-    python3 python3-numpy python3-scipy python3-h5py python3-pip \
```

Comment:
please remove the pip command and the ENV PATH command at the bottom of this file